app/src/main/java/com/blabbertabber/blabbertabber/TheAudioRecord.java

```diff
@@ -0,0 +1,89 @@
+package com.blabbertabber.blabbertabber;
+
+import android.media.AudioFormat;
+import android.media.AudioRecord;
+import android.util.Log;
+
+/**
+ * Created by Cunnie on 10/8/15.
+ * http://www.javaworld.com/article/2073352/core-java/simply-singleton.html
+ */
+
+// BUGS: uses 'static' which makes testing much more difficult, for example we don't test that
+//    release() calls MediaRecorder.reset() instead of MediaRecorder.release();
+
+public class TheAudioRecord extends AudioRecord {
+    private static final String TAG = "TheAudioRecord";
+    private static final int RECORDER_AUDIO_SOURCE = BestMicrophone.getBestMicrophone();
+    // http://developer.android.com/reference/android/media/AudioRecord.html
+    // "44100Hz is currently the only rate that is guaranteed to work on all devices"
+    // 16k samples/sec * 2 bytes/sample = 32kB/sec == 115.2 MB/hour
```

Comment:
So we're doing only roughly a third of the only rate that is guaranteed to work on all devices.